test/testdata/input/splicelist_in_nested_block_list.test

```diff
@@ -0,0 +1,7 @@
+SLICE LIST IN NESTED BLOCK LIST
+---
+key:
+- true
+- - false
+---
+- [splice, [key, 1], 0, 1, ['test']]
```

Comment:
Missing newline at end of the line.

Consider making a configuration change to your editor.